### ISSUE TITLE
Binary update-secureboot-policy not found, modules won't be signed

### ISSUE THREAD

**Issue by loveisfoss** [dell/dkms#254]
I have a secure boot enabled system (Debian bookworm) and installed nvidia-drivers 470. Installed dkms version is 3.0.6. Howerver, when installing nvidia-driver and auto-building with dkms, the modules are not getting signed.

When I try to build the module myself, I get the following error:
```
# dkms build nvidia-current/470.141.03
Sign command: /usr/lib/linux-kbuild-5.19/scripts/sign-file
Signing key: /var/lib/dkms/mok.key
Public certificate (MOK): /var/lib/dkms/mok.pub
Binary update-secureboot-policy not found, modules won't be signed
...
```
The modules are getting built but are not getting signed.
This happens both when using "sudo" before dkms and when executing the command as root (after doing sudo su).

I would like to refer to line 892 of dkms.in:
```
if [[ ! -f "${mok_signing_key}" ]] || [[ ! -f "${mok_certificate}" ]]; then
                    if [[ ! -x "$(command -v update-secureboot-policy)" ]]; then
                        echo "Binary update-secureboot-policy not found, modules won't be signed"
                        return
                    fi
                    # update-secureboot-policy won't create new key if $mok_certificate exists
                    if [[ -f "${mok_certificate}" ]]; then
                        rm -f "${mok_certificate}"
                    fi
                    echo "Certificate or key are missing, generating them using update-secureboot-policy..."
                    SHIM_NOTRIGGER=y update-secureboot-policy --new-key &>/dev/null
                fi
```

I tried running "command -v update-secureboot-policy" and I get no output.
However, when running as root (sudo su), I get an answer, and it is as follows:
```
# command -v update-secureboot-policy
/usr/sbin/update-secureboot-policy
```
Furthermore, I have put my custom certificate and key in the default location:
```
ls -l /var/lib/dkms/
total 16
-rwxr-xr-x 1 root root 1700  1 okt 14.30 mok.key
-rwxr-xr-x 1 root root  825  1 okt 14.30 mok.pub
drwxr-xr-x 3 root root 4096  1 okt 15.38 nvidia-current
drwxr-xr-x 3 root root 4096  1 okt 12.18 xone
```
The certificate is of course enrolled in UEFI setup. The automated key signing was happening without problems some time ago, before the signing process rework (not complaining, just trying to explain).

Any ideas?

**Comment by loveisfoss** [comment]
This was solved by downloading the code for dkms 3.0.7, compiling it and trying again.
Version 3.0.7 is not available in Debian testing/sid in the time of writing.

**Comment by loveisfoss** [comment]
This was solved by downloading the code for dkms 3.0.7, compiling it and trying again.
Version 3.0.7 is not available in Debian testing/sid in the time of writing.
